### ISSUE TITLE
How to debug/view live/computed styles in Chrome dev tools?

### ISSUE THREAD

**Issue by jasonkylefrank** [FormidableLabs/radium#532]
Hello,

I'm fairly new to React and Radium.  I cloned this repo so I could work with the examples.  The problem I'm having is figuring out how to get **Chrome dev tools** let me work with the actual live/computed styles (screenshots will follow which explain specifics).  

I'm wondering if this is a virtual DOM issue that Chrome is not setup to work with yet?  Is there something I can do to solve these issues?

Here's what I'm talking about.  

**Viewing and working with basic styles:**

![image](https://cloud.githubusercontent.com/assets/7502365/12373697/8c734bc6-bc4f-11e5-8f0f-65b62af63d25.png)

![image](https://cloud.githubusercontent.com/assets/7502365/12373699/c2866252-bc4f-11e5-8949-70565149b48b.png)

![image](https://cloud.githubusercontent.com/assets/7502365/12373701/d7f3233c-bc4f-11e5-91dd-ff57c57c8edc.png)

<br />
**Viewing and working with Media Queries** (3 screenshot series)

![image](https://cloud.githubusercontent.com/assets/7502365/12373702/ea7e56b6-bc4f-11e5-846c-df5b282f6973.png)

![image](https://cloud.githubusercontent.com/assets/7502365/12373706/13615c86-bc50-11e5-9fc6-543687323fb0.png)

...but I can't get those media query styles to show up in the "Styles" or "Computed" tabs:

![image](https://cloud.githubusercontent.com/assets/7502365/12373708/18d07d14-bc50-11e5-9ec5-638fa97fc4c6.png)

Thanks for you help!

**Comment by jasonkylefrank** [comment]
UPDATE:

After a full system reboot Chrome dev tools seem to be working Ok with the styles.  Note that before rebooting, page refreshes did not solve the issue.  

One further piece of debugging information:  I have the Chrome React tools installed, but when I run your examples **the 'React' tab does not appear** (React is not detected).  You can see this in my above screenshots.  However, if I open the example via `http://localhost:8080/` instead of `http://localhost:8080/webpack-dev-server/`, then it properly detects React and provides the 'React' tab in the Chrome dev tools.  However, as expected, when I run it from `http://localhost:8080/` I do not get hot-reloading (due to Webpack's config).   

So in summary about the 'React' tab:
1. This appears to be a separate problem from my originally-reported styles debugging issue
2. I'm guessing this issue is a Webpack setup problem

Again, the original styles-debugging issue appears to be working now after a full system reboot.  Anyone else ever had this problem?  Does it come-and-go?

**Comment by ianobermiller** [comment]
I haven't seen this myself, but thank you for the detailed report, love the screenshots. So is the only remaining issue that the React tab does not show up in the examples? This is probably just an artifact of our setup, but we should definitely fix it anyway.

**Comment by jasonkylefrank** [comment]
@ianobermiller Yeah, the 'React' tab issue appears to be the only remaining issue.  (But I was curious to hear from others to see if they experience this issue from time to time.).

**Comment by alexlande** [comment]
I haven't experienced the original issue with Radium in particular, but I see it semi-regularly with normal CSS and with inline styles.

**Comment by jasonkylefrank** [comment]
@alexlande Thanks, good to hear your feedback.

**Comment by daniloprates** [comment]
Regarding this subject I wondering if there's a way to display the properties like JS objects instead of CSS... does any one know if it's possible?
